app/controllers/documents_controller.rb

```diff
@@ -23,6 +23,16 @@ def show
     @edition = Edition.find_current(document: params[:document])
   end
 
+  def history
+    @edition = Edition.find_current(document: params[:document])
+    @timeline_entries = @edition.document.timeline_entries
+                                  .includes(:created_by, :details)
+                                  .order(created_at: :desc)
+                                  .includes(:edition)
+                                  .page(params.fetch(:page, 1))
+                                  .per(25)
```

Comment:
Indentation seems a little off here. I also wonder if we want to query TimelineEntry directly for example via `TimelineEntry.where(document: @edition.document)` and then do the chain?